stories/Dropdown.js

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import Dropdown from '../src/components/Dropdown';
+import DropdownItem from '../src/components/DropdownItem';
+
+import defaultTheme from '../src/themes/DefaultTheme';
+import darkTheme from '../src/themes/DarkTheme';
+
+import {
+  NewWindowIcon,
+  DownloadIcon,
+  ExpireTimeIcon,
+  PadlockLockIcon,
+  PadlockUnlockIcon,
+  TeamIcon,
+  TrashBinIcon,
+} from '../src/components/icons';
+
+storiesOf('Dropdown', module)
+  .addWithJSX('Dropdown minimum props', () => {
+    function onItemClick(e) {
+      console.log('click');
+    }
+
+    return (
+      <Dropdown isActive>
+        <DropdownItem title="Dropdown item 1" onClick={onItemClick} active />
```

Comment:
Because the `onClick` prop is set to an empty function by default, we don't need to pass anything to it here ;)